src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -372,11 +359,10 @@ internal IEnumerable<V> Search(string query, int minResultsForTolerantSearch = 0
             var orderedSearchDict = searchDict.OrderByDescending(x => x.Value);
 
             var searchResults = orderedSearchDict.Select(x => x.Key);
-            if (TruncateSearchResults)
-            {
-                searchResults = searchResults.Take(20);
-            }
 
+            // return only the top 20 search results
+            searchResults = searchResults.Take(20);
+            
 #if DEBUG
```

Comment:
@ColinDayOrg I'm assuming we want to leave the logging work you did?

Reply:
I think so, it's not a lot of logging output and the output could be useful.

Reply:
Odd that file comments don't show up here always and vice versa, so repeating here, it's not a lot of  logging output and it could be useful (and is only under debug anyways) so I would say yes.